src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -1252,20 +1252,23 @@ class NodeConfig(object):
             # Huge page memory
             host_hpmem = self.get_hpmem_on_node(ignore_reserved=True)
             host_resv_hpmem = host_hpmem - self.get_hpmem_on_node()
-            host_resv_hpmem = 0 if (host_resv_hpmem < 0) else host_resv_hpmem
+            if (host_resv_hpmem < 0):
```

Comment:
Python style doesn't use () in ifs unless necessary.  Please remove them (and for the other cases in this function)

Reply:
Done.